SConstruct

```diff
@@ -583,76 +584,6 @@ print(f'Building for platform "{selected_platform}", architecture "{env["arch"]}
 if env.dev_build:
     print("NOTE: Developer build, with debug optimization level and debug symbols (unless overridden).")
 
-# Set optimize and debug_symbols flags.
-# "custom" means do nothing and let users set their own optimization flags.
-# Needs to happen after configure to have `env.msvc` defined.
-if env.msvc:
-    if env["debug_symbols"]:
-        env.Append(CCFLAGS=["/Zi", "/FS"])
-        env.Append(LINKFLAGS=["/DEBUG:FULL"])
-    else:
-        env.Append(LINKFLAGS=["/DEBUG:NONE"])
```

Comment:
Technically I just moved the compiler version checks _before_ this stuff, but Git shows it the other way around ;)